web_programming/fetch_github_info.py

```diff
@@ -36,6 +36,8 @@
 def fetch_github_info(auth_token: str) -> dict[Any, Any]:
     """
     Fetch GitHub info of a user using the requests module
```

Comment:
Is that enough or do we need more tests?

Reply:
You're right, added much more for anyone that doesn't know the requests library